es-vue-base/src/lib-components/EsFooter.vue

```diff
@@ -0,0 +1,158 @@
g/2000/svg" width="160" height="36" fill="none" viewBox="0 0 160 36"><g clip-path="url(#a)"><path fill="#616161" d="M2.3 19.2c0 3.7 1 4.5 2.8 4.5 1.4 0 2.2-1.2 2.3-2.6h2.4c0 3-1.9 4.4-4.7 4.4-2.8 0-5.1-.9-5.1-6.8 0-4 .5-7.2 5.1-7.2 3.8 0 4.8 2 4.8 6.5v1.2H2.3Zm5.3-1.7c0-3.9-1.1-4.4-2.7-4.4-1.4 0-2.5.7-2.6 4.4h5.3ZM20.8 25.2v-9.3c0-1.5-.5-2.6-2.4-2.6-2.4 0-2.7 2-2.7 4v7.9h-2.2V14.5l-.1-2.8h2.3v1.8c1-1.5 2-2 3.7-2 2.8 0 3.6 1.7 3.6 4.4v9.3h-2.2ZM29 19.2c0 3.7 1 4.5 2.6 4.5 1.5 0 2.3-1.2 2.4-2.6h2.4c0 3-1.9 4.4-4.7 4.4-2.8 0-5.1-.9-5.1-6.8 0-4 .4-7.2 5-7.2 4 0 4.9 2 4.9 6.5v1.2h-7.6Zm5.2-1.7c0-3.9-1.1-4.4-2.7-4.4-1.4 0-2.6.7-2.6 4.4h5.3ZM42.3 13.8c1-2 2.1-2.3 4.2-2.3v2.2h-1.1c-2.5 0-3 1.7-3 3.6v7.9h-2.2V11.7h2.1v2ZM56 11.7h2.2v14.8c0 3-2 3.8-5 3.8-2.2 0-4.2-1.2-4-3.6h2.4c0 1.2.9 2 2 2 1.6 0 2.4-1 2.4-2.4v-2.8c-.6 1-1.8 1.7-3 1.7-3.7 0-4.4-3.2-4.4-7.2 0-6.1 3.1-6.5 4.2-6.5 1.4 0 2.5.6 3.2 1.9v-1.7m-2.5 1.6c-2.5 0-2.6 3-2.6 4.7 0 4.4 1 5.4 2.6 5.4 2.7 0 2.6-3.1 2.6-5 0-2 .1-5.1-2.6-5.1ZM66.2 22.7l3.3-11H72l-6.1 18.6h-2.3l1.5-5.1-4.5-13.5H63l3.2 11Z"/><path fill="#CA6F20" d="M78 25.5c-3 0-4.8-1.3-4.7-4.4h2.5c0 .8 0 2.6 2.2 2.6 1.3 0 2.3-.7 2.3-2 0-2.5-6.7-2.5-6.7-6.6 0-1.4.9-3.6 4.7-3.6 2.4 0 4.4 1.1 4.2 3.8h-2.4c0-1.4-.7-2.2-2-2.2s-2.2.7-2.2 1.8c0 2.4 6.7 2.4 6.7 6.5 0 3-2 4-4.6 4ZM92.7 23.2c-.7 1.6-1.8 2.3-3.4 2.3-2.9 0-3.7-1.9-3.7-4.5 0-4 4-4.2 7-4.1.1-1.8.1-3.8-2.2-3.8-1.5 0-2.2 1-2.1 2.4h-2.4c.1-3 1.7-4 4.6-4 3.4 0 4.3 1.8 4.3 4V22c0 1 .1 2.2.3 3.2h-2.4v-2Zm-5-2c0 1.3.7 2.5 2.2 2.5 1.3 0 3-.9 2.7-5.2-2 0-4.8-.2-4.8 2.7ZM106.2 11.7h2.1v14.8c0 3-2 3.8-4.9 3.8-2.2 0-4.2-1.2-4-3.6h2.4c0 1.2.8 2 2 2 1.5 0 2.4-1 2.4-2.4v-2.8h-.1c-.6 1-1.8 1.7-3 1.7-3.6 0-4.3-3.2-4.3-7.2 0-6.1 3-6.5 4.2-6.5 1.4 0 2.5.6 3.1 1.9v-1.7m-2.5 1.6c-2.5 0-2.5 3-2.5 4.7 0 4.4 1 5.4 2.6 5.4 2.6 0 2.5-3.1 2.5-5 0-2 .2-5.1-2.6-5.1ZM114.5 19.2c0 3.7 1 4.5 2.7 4.5 1.5 0 2.3-1.2 2.3-2.6h2.4c0 3-1.8 4.4-4.6 4.4-2.9 0-5.2-.9-5.2-6.8 0-4 .5-7.2 5.2-7.2 3.8 0 4.7 2 4.7 6.5v1.2h-7.5Zm5.2-1.7c0-3.9-1-4.4-2.7-4.4-1.3 0-2.5.7-2.5 4.4h5.2Z"/><path fill="#616161" d="M154 12c-.9-1.7-2.3-3.5-4-5.4-2.8-3.2-5.6-5.6-5.7-5.7a2.2 2.2 0 0 0-2.8 0 62.2 62.2 0 0 0-9.8 11c-1.4 2.4-2 4.4-2 6.2 0 7 5.3 12.6 12 13.2v3.3c0 .7.6 1.2 1.2 1.2s1-.5 1-1.2v-3.3c6.8-.6 12.2-6.3 12.2-13.2 0-1.8-.7-3.8-2-6.1ZM143.5 29h-1a11 11 0 0 1-10.5-11c0-6 11-15.5 11-15.5s11 9.4 11 15.5a11 11 0 0 1-10.5 11Z"/><path fill="#616161" d="M147.3 15.9a1 1 0 0 0-1-.4l-1.8.3.4-7.7a1 1 0 0 0-1.8-.5l-4.7 11.1a1 1 0 0 0 0 1c.2.4.6.6 1 .5l1.8-.3L140 26a1 1 0 0 0 1 1h.1c.3 0 .5-.3.7-.5l5.5-9.7a1 1 0 0 0 0-1m-1.2.8-4.7 8.3 1-6.5-2.9.5 4.2-9.7-.4 7.9 2.8-.5ZM158.6 22.6a1.4 1.4 0 0 0-1.5 1.5c0 .8.7 1.4 1.5 1.4.7 0 1.4-.6 1.4-1.4 0-.8-.7-1.5-1.4-1.5Zm1.1 1.5c0 .6-.5 1.1-1.1 1.1-.7 0-1.2-.5-1.2-1.1 0-.7.5-1.2 1.2-1.2.6 0 1.1.5 1.1 1.2Z"/><path fill="#616161" d="m159 23.8-.1-.3-.4-.1h-.4v1.3h.2v-.5h.2l.4.5h.2l-.4-.6c.2 0 .3-.1.3-.3Zm-.5.2h-.2v-.4h.4v.4h-.2Z"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 .4h160v35.3H0z"/></clipPath></defs></svg>
```

Comment:
I'm concerned about the IDs and URLs in the SVGs.

It's a pain to hand-edit SVG code, but since these footer SVGs will be included in every page we should [prefix them](https://github.com/EnergySage/es-site/blob/develop/docs/frontend/svg.md#process) (with something like `footer-`?) to avoid conflicting with other SVGs.

I think it's inevitable that some other exported SVG will use the same `#a`. What do you think?

Reply:
This specific SVG is the EnergySage typeface+leaf which doesn't exist in the es-ds yet but will after that gets merged. I was thinking it would be a non-blocking fast follow to replace with the actual logo but could prioritize it if that makes sense: https://energysage.atlassian.net/browse/CED-413